src/Uno.UI.Tests/Windows_UI_Xaml_Data/xBindTests/Controls/Functions_Control.xaml

```diff
@@ -14,13 +14,13 @@
 				   Text="{x:Bind InstanceProperty}" />
 		<TextBlock x:Name="_StaticProperty"
 				   x:FieldModifier="public"
-				   Text="{x:Bind StaticProperty}" />
+				   Text="{x:Bind local:Functions_Control.StaticProperty}" />
```

Comment:
This is a scenario that is supported by WinUI, if I'm not mistaken. It would be best to add a separate test for this.

Reply:
@jeromelaban I think it's not, unless I have done a wrong test.

Reply:
Indeed, it's not:
![image](https://user-images.githubusercontent.com/5839577/220122842-73298e50-e935-44bc-9a62-71bcf4319aea.png)
![image](https://user-images.githubusercontent.com/5839577/220122804-a80cf6fd-d5f2-4806-b9e0-fc34550c841a.png)

Reply:
![image](https://user-images.githubusercontent.com/31348972/220122892-178aad2d-8e40-43d8-b39a-37c24222d223.png)